### PR TITLE
feat: alert notify script

### PR DESCRIPTION
**Pull request by jjyr**

### CODE REVIEW

File: util/network-alert/src/alert_relayer.rs [hunk 1]
```diff
@@ -33,9 +33,9 @@ pub struct AlertRelayer {
 }
 
 impl AlertRelayer {
-    pub fn new(client_version: String, config: Config) -> Self {
+    pub fn new(client_version: String, notifier_config: NotifierConfig, config: Config) -> Self {
```
Comment (keroro520) [inline util/network-alert/src/alert_relayer.rs]: Why not merge them together?  Put `client_version` and `notify_script` into config when constructs. And `AlertRelayer` and `Notifier` could share the same config.

File: ckb-bin/src/subcommand/run.rs [hunk 2]
```diff
@@ -67,8 +67,13 @@ pub fn run(args: RunArgs, version: Version) -> Result<(), ExitCode> {
 
     let relayer = Relayer::new(chain_controller.clone(), sync_shared_state);
     let net_timer = NetTimeProtocol::default();
-    let alert_config = args.config.alert.unwrap_or_default();
-    let alert_relayer = AlertRelayer::new(version.to_string(), alert_config);
+    let alert_signature_config = args.config.alert_signature.unwrap_or_default();
```
Comment (doitian) [inline ckb-bin/src/subcommand/run.rs]: `serde(default)` can be used if we always want something.

File: util/network-alert/src/notifier.rs [hunk 3]
```diff
@@ -60,13 +64,30 @@ impl Notifier {
...
         self.noticed_alerts
             .sort_by_key(|a| std::u32::MAX - a.priority);
     }
 
+    fn notify(&self, alert: &Alert) {
+        warn!("receive a new alert: {}", alert.message);
+        if let Some(notify_script) = self.config.notify_script.as_ref() {
+            match Command::new(notify_script)
+                .args(&[alert.message.to_owned()])
+                .status()
```
Comment (doitian) [inline util/network-alert/src/notifier.rs]: The child process may never exit.

Comment (doitian) [reply util/network-alert/src/notifier.rs]: Not sure this still works https://github.com/alexcrichton/wait-timeout

Comment (jjyr) [reply util/network-alert/src/notifier.rs]: I think we should make it simple, user can set a timeout in his own customized script(IMO it's unnecessary in 99% cases) and decide how to exit child process which better than we kill the script.

**Pull request merged**